src/components/alerts/alert.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import bindAll from 'lodash.bindall';
+
+import Box from '../box/box.jsx';
+import Button from '../button/button.jsx';
+
+import styles from './alert.css';
+
+class AlertComponent extends React.Component {
+    constructor (props) {
+        super(props);
+        bindAll(this, [
+            'handleOnCloseAlert'
+        ]);
+    }
+    handleOnCloseAlert () {
```

Comment:
@rschamp I tried to put this handler in the container, but I found no lint-friendly way to handle passing up the index property to the parent container (no `bind` or arrow functions allowed in the `onClick`).  After lots of Googling, I ended up at this pattern, but let me know if there is a better way to handle this click function in the container somehow.  Thanks!

Reply:
Ah, I see! One level of abstraction I wasn't seeing, thank you!